examples/PublishHelloWorld/main.c

```diff
@@ -26,18 +26,21 @@
 
 int main(int args, char** argv)
 {
-    if(args >= 2 && (0 == strcmp("-h", argv[1]) || 0 == strcmp("--help", argv[1]) || 0 == atoi(argv[1])))
+    // CLI
+    if(3 > args || 0 == atoi(argv[2]))
     {
-        printf("usage: program [-h | --help | <topics>]\n");
+        printf("usage: program [-h | --help] | ip port [<topics>]\n");
```

Comment:
Just a suggestion, we could use `<number-of-publications>` instead of `<topics>` in order to improve usability.

Reply:
I like to change the name of `topics`, but... what about something more little as `amount` or `quantity`?

Reply:
Ok perfect, whatever you like. Even you could keep the current one.

Reply:
After an existential discussion with myself, I decided to name it `max_topics`

Reply:
xD, perfect!